backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/filters/postalcode/PostalCodeRecord.java

```diff
@@ -0,0 +1,10 @@
+package com.bakdata.conquery.models.datasets.concepts.filters.postalcode;
+
+import lombok.Data;
+
+@Data
+public class PostalCodeRecord {
+	final private int plz1;
+	final private int plz2;
```

Comment:
Persönlich eher fan von left/right als 1/2. Mag aber einfach variabelnamen mit Zahlen drin nicht.

Reply:
oder from/to